src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/PurchaseDialog/index.js

```diff
@@ -42,7 +42,7 @@ const PurchaseDialog = ({ classes, show, onClose, startDepositLoader, startWithd
     try {
       const amountInAGI = amount[txnTypes.DEPOSIT];
       const amountInCogs = agiToCogs(amountInAGI);
-      const response = await sdk.account.depositToEscrowAccount(amountInCogs);
+      await sdk.account.depositToEscrowAccount(amountInCogs);
       setAmount({});
```

Comment:
Why do you need to set the amount here?

Reply:
I'm resetting the amount once it is deposited.